### PR TITLE
Fixed issue 19509 - Catalog bundle product Bundle Items inconsistent box border

### PR DESCRIPTION
**Pull request by abrarpathan19**
### Description (*)

 Catalog bundle product Bundle Items inconsistent box border

### Fixed Issues (if relevant)

1. magento/magento2#19509:  Catalog bundle product Bundle Items inconsistent box border

### Manual testing scenarios (*)

1. N/A

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @abrarpathan19. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

File: app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/main/_collapsible-blocks.less [hunk 1]
```diff
@@ -147,6 +147,11 @@
     &.collapsible-block-wrapper-last {
         border-bottom: 0;
     }
+    .admin__dynamic-rows.admin__control-collapsible{
```
Comment (sivaschenko) [inline app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/main/_collapsible-blocks.less]:
```suggestion
    .admin__dynamic-rows.admin__control-collapsible {   
```

**Review by sivaschenko — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko, thank you for the review.
[ENGCOM-3605](https://jira.corp.magento.com/browse/ENGCOM-3605) has been created to process this Pull Request

**Comment by magento-engcom-team** [comment]
Hi @abrarpathan19. Thank you for your contribution. 
We will aim to release these changes as part of 2.3.1. 
Please check the release notes for final confirmation.

**Pull request merged**